Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/cryoscope_amplitude_calibration.py

```diff
@@ -0,0 +1,205 @@
+"""
+cryoscope_calib.py: template for performing the detuning vs flux pulse amplitude calibration prior to the cryoscope
+protocol.
+Author: Theo Laudat - Quantum Machines
+Created: 02/05/2022
+"""
+from qm.qua import *
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm import SimulationConfig, LoopbackInterface
+from configuration import *
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+###################
+# The QUA program #
+###################
+
+n_avg = 10000  # Number of averages
+cooldown_time = 5 * qubit_T1 // 4  # Cooldown time in clock cycles (4ns)
+# Flux amplitude sweep (as a prefactor of the flux amplitude)
+n_flux_amp = 401
+flux_amp_array = np.linspace(0, -0.2, n_flux_amp)
+
+# Macro for measuring the averaged ground and excited states for calibration
+def ge_averaged_measurement(n_avg_cal):
```

Comment:
Not sure exactly why it is needed, but if it is used in multiple files, then it should either go to the config, to py_qua_tools, or to a separate macro file